Description: Change the SOVERSION, to revert an useless/wrong/unintentional
 bump made by upstream in https://bazaar.launchpad.net/~stewart/libeatmydata/main/revision/86
 Also bumping the revision, due to the code change.
Author: Mattia Rizzolo <mattia@mapreri.org>
Forwarded: no
Last-Update: 2015-03-18


--- a/Makefile.am
+++ b/Makefile.am
@@ -81,7 +81,7 @@
 	-DBUILDING_LIBEATMYDATA
 
 libeatmydata_la_LIBADD = $(LIBDL_LIBS) -ldl
-libeatmydata_la_LDFLAGS = $(AM_LDFLAGS) -avoid-version
+libeatmydata_la_LDFLAGS = $(AM_LDFLAGS) -version-info $(LIBEATMYDATA_VERSION)
 
 #install-exec-hook:
 #	find $(DESTDIR)$(libdir) -type f -name \*.la |xargs rm -f
--- a/configure.ac
+++ b/configure.ac
@@ -16,6 +16,33 @@
 AC_CONFIG_AUX_DIR([config])
 AC_CONFIG_MACRO_DIR([m4])
 
+# Increment if interfaces have been added, removed, or changed.
+LIBEATMYDATA_VERSION_CURRENT=4
+# Increment if source code has changed, set to zero if current is incremented.
+LIBEATMYDATA_VERSION_REVISION=0
+# Increment if interfaces have been added, set to zero if interfaces have been
+# removed or changed.
+LIBEATMYDATA_VERSION_AGE=3
+
+AC_SUBST(LIBEATMYDATA_VERSION_CURRENT)
+AC_SUBST(LIBEATMYDATA_VERSION_REVISION)
+AC_SUBST(LIBEATMYDATA_VERSION_AGE)
+
+AC_DEFINE_UNQUOTED([LIBEATMYDATA_VERSION_CURRENT],
+                   [$LIBEATMYDATA_VERSION_CURRENT],
+                   [Current portion of liblibeatmydata version])
+AC_DEFINE_UNQUOTED([LIBEATMYDATA_VERSION_REVISION],
+                   [$LIBEATMYDATA_VERSION_REVISION],
+                   [Revision portion of liblibeatmydata version])
+AC_DEFINE_UNQUOTED([LIBEATMYDATA_VERSION_AGE],
+                   [$LIBEATMYDATA_VERSION_AGE],
+                   [Age portion of liblibeatmydata version])
+
+LIBEATMYDATA_VERSION=$LIBEATMYDATA_VERSION_CURRENT:$LIBEATMYDATA_VERSION_REVISION:$LIBEATMYDATA_VERSION_AGE
+
+AC_SUBST(LIBEATMYDATA_VERSION)
+
+
 AC_CANONICAL_TARGET
 AC_ARG_PROGRAM
 AC_USE_SYSTEM_EXTENSIONS
